config/sync/core.entity_view_display.node.stanford_news.default.yml

```diff
@@ -111,6 +116,31 @@ third_party_settings:
                     class: su-news-title
             weight: 4
             additional: {  }
+          1629bc10-f79b-4aae-9568-b34a981c24c7:
+            uuid: 1629bc10-f79b-4aae-9568-b34a981c24c7
+            region: main
+            configuration:
+              id: 'field_block:node:stanford_news:su_news_variant'
```

Comment:
From our 1:1

Replace the current select `su_news_variant` with the Layout.
- Create Spotlight News layout
- Enable layout in News Display Mode
- Move field to the top
- Update dependencies